### PR TITLE
update yanglalo to v2

### PR DESCRIPTION
**Pull request by LinguList**
please see this example for how to use the form spec if you want to avoid that a splitting will yield ALL values, but rather prefer to have only the first value in the data.

Please also check this in order to see how to make use of in-built aspects, like csv-readers (no need to import the csv module here).

### CODE REVIEW

File: lexibank_yanglalo.py [hunk 1]
```diff
@@ -1,82 +1,72 @@
...
     dir = Path(__file__).parent
-    concept_class = HConcept
-    language_class = HLanguage
-
-    def cmd_download(self, **kw):
-        # nothing to do, as the raw data is in the repository
-        pass
+    concept_class = CustomConcept
+    language_class = CustomLanguage
+    form_spec = FirstFormOnlySpec(missing_data=('烂饭', '-'))
```
Comment (LinguList) [inline lexibank_yanglalo.py:28]: This is the important point. If you want to split the first entry off, use the FirstFormOnlySpec.

File: lexibank_yanglalo.py [hunk 2]
```diff
@@ -1,82 +1,72 @@
...
-
-    def cmd_install(self, **kw):
+    def cmd_makecldf(self, args):
 
         # read raw data for later addition
-        filename = self.dir.joinpath("raw", "raw_data.tsv").as_posix()
-        with open(filename) as csvfile:
-            reader = csv.DictReader(csvfile, delimiter="\t")
-            raw_entries = [row for row in reader]
+        raw_entries = self.raw_dir.read_csv("raw_data.tsv", dicts=True, delimiter="\t")
```
Comment (LinguList) [inline lexibank_yanglalo.py:33]: use in-built dictionary readers for csv.

File: lexibank_yanglalo.py [hunk 3]
```diff
@@ -1,82 +1,72 @@
...
 
         # add information to dataset
-        with self.cldf as ds:
-            ds.add_sources(*self.raw.read_bib())
+        args.writer.add_sources()
```
Comment (LinguList) [inline lexibank_yanglalo.py:36]: no need to read courses, just leave it emtpy, it searches for the sources in `raw`.

File: lexibank_yanglalo.py [hunk 4]
```diff
@@ -1,82 +1,72 @@
...
-                            ds.add_cognate(
-                                lexeme=row,
-                                Cognateset_ID=cogid,
-                                Source=["Yang2011Lalo"],
-                                Alignment_Source="",
+                            Cognacy=cogid
+                            ):
+                        args.writer.add_cognate(
+                            lexeme=row,
+                            Cognateset_ID=cogid+1,
```
Comment (LinguList) [inline lexibank_yanglalo.py:70]: I suspect, enumerate starts with 0, we don't want a 0 as an ide for a cognate, right?

**Comment by chrzyki** [comment]
Merging this for now to test with updated CLICS.

**Pull request merged**